bowling/swift/Bowling/README.md

```diff
@@ -14,19 +16,26 @@ In the tenth frame a player who rolls a spare or strike is allowed to roll the e
 balls to complete the frame.  However no more than three balls can be rolled in
 tenth frame.
 
-Original Source: http://butunclebob.com/files/downloads/Bowling%20Game%20Kata.ppt
+Sources:
+- [kata logs](http://kata-log.rocks/bowling-game-kata)
+- [Power point](http://butunclebob.com/files/downloads/Bowling%20Game%20Kata.ppt)
 
-____
+## How to start:
 
-1. Write the first failing test. Then write a factory that returns an object, or an aggregate of objects, that make the test pass.
+1. Write the first failing test. Then write code to make the first test pass.
 
-The factory should be limited to creating objects and linking them together. No conditionals allowed.
+The code written should only be intended to make the first test pass.
```

Comment:
You should indent this at the same level of the `Write` on the previous line, so it shows inside the bullet point, instead of being outside of it (same for all "sub-items" below)